mithril-aggregator/src/snapshotter.rs

```diff
@@ -174,4 +231,71 @@ mod tests {
             )
         );
     }
+
+    #[test]
+    fn should_create_directory_if_does_not_exist() {
```

Comment:
In this function and below, no need to set the `// arrange`, `// act` and `// assert`